test/test_property.py

```diff
@@ -19,11 +19,11 @@ def test_bool_conversion(self):
         assert(p.dtype == 'boolean')
         assert(p.value == [True, False, True, False, True])
 
-        q = Property(name='sent', value=['False', True, 'TRUE', '0'])
+        q = Property(name='sent', value=['False', True, 'TRUE', '0', 't', 'F', 'Ft'])
```

Comment:
Right, I was thinking of the `boolean_get` method that checks for specific strings.